packages/juggler/examples/controllers/CustomerController1.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/juggler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {EntityCrudRepository} from '../../src/repository';
+import {repository} from '../../src/decorator';
+import {Customer} from '../models/customer';
+
+/**
+ * Controller for Customer
+ */
+// @controller
+export class CustomerController1 {
```

Comment:
The difference between `CustomerController1`, `CustomerController2` and `CustomerController3` are difficult to tell by looking at the names. Could you please use more descriptive names, e.g. `ControllerWithPropertyDI` and `ControllerWithConstructorDI`?

We also agreed to use kebab case for file names, while these two example files are using PascalCase. Is that intentional?